containers/pre_built_workers/php7/run_worker.sh

```diff
@@ -13,6 +13,25 @@
 # limitations under the License.
 
 set -ex
+
```

Comment:
In a follow-up PR, perhaps you can run  "shellcheck" on all shell scripts and fix outstanding issues. For now,  I am approving it as-is.

Reply:
Thank you for reminding me this, I was not able to use shellcheck earlier, now I have figured out how use use it now! The only thing is "source /usr/local/rvm/scripts/rvm" is not a local directory, shellcheck shows it as invalid path, how should I fix this?